src/strategies/cap-voting-power/index.ts

```diff
@@ -0,0 +1,267 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Contract } from '@ethersproject/contracts';
+
+export const author = 'unRealGamer28';
+export const version = '0.1.0';
+
+const abi = ['function totalSupply(uint256 t) external view returns (uint256)'];
+
+const maxLockupDuration = 4 * 365 * 24 * 60 * 60; // Maximum lockup duration is 4 years in seconds based on Curve VE contract
+const maxVestingDuration = 4 * 365 * 24 * 60 * 60; // Maximum vesting duration is 4 years in seconds
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  // Assertions/checks to validate options
+  options.vestingAddresses.forEach((vestingAddress) => {
```

Comment:
Hey @unRealGamer28 can you add a limit to `vestingAddresses` so it doesn't use all memory 😃